fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1215,13 +1215,17 @@ create_stmt ::=
     :}
     ;
 
-opt_aggregate ::=
+opt_function_type ::=
     {:
-        RESULT = false;
+        RESULT = null;
     :}
-    | KW_AGGREGATE
+    | KW_TABLES:table
```

Comment:
why add "s"? KW_TABLE ?

Reply:
`CREATE TABLE function` may has conflict with `create table table_name ()`. 
I haven't found a way to resolve the conflict yet.

Reply:
resolved